matchMinima.py

```diff
@@ -432,13 +395,15 @@ def getRatioTimes(allMolTimes, zeroes):
 
     """
 
+#M Unrelated: You're compring times spent in each minima? What for?
```

Comment:
i'm taking into acct how long it takes to optimize each conformer of a molecule. then getting an average per molecule per level of theory. that's what's being compared, the diff levels of theory